BlogTemplate/Pages/Drafts.cshtml.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using BlogTemplate.Models;
+using System.IO;
+
+namespace BlogTemplate.Pages
+{
+    public class DraftsModel : PageModel
+    {
+        const string StorageFolder = "BlogFiles";
+
+        private Blog _blog;
+        private BlogDataStore _dataStore;
+
+        public DraftsModel(Blog blog, BlogDataStore dataStore)
```

Comment:
do we use _blog for anything?  I'm pretty sure we don't actually use the Blog class anywhere